docs/src/modules/components/AppSearch.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { createPortal } from 'react-dom';
```

Comment:
I'd replace lines like this one with a namespace import as well to be consistent.

Reply:
Ah I missed these kinds of imports, let me see if eslint can find them too